lib/ui/screens/UserInfoCollectorScreen.dart

```diff
@@ -78,155 +80,40 @@ class _UserInfoCollectorScreenState extends BaseState<UserInfoCollectorScreen> {
-              ),
-            ),
-          ),
-          alignment: Alignment.center,
-        ),
-        SizedBox(
-          height: 10,
-        ),
-        Align(
-          child: SizedBox(
-            width: double.infinity,
-            child: MaterialButton(
-              color: Colors.green,
-              height: 50,
-              shape: RoundedRectangleBorder(
-                borderRadius: new BorderRadius.circular(8.0),
-              ),
-              onPressed: () async {
-                dialogOnResponse(2);
-                FirestoreRepository.setUserSeverity(0);
-              },
-              child: Text(
-                "I am not infected with Coronavirus",
-                textAlign: TextAlign.center,
-                style: TextStyle(color: Colors.white, fontSize: 18),
-              ),
-            ),
-          ),
-          alignment: Alignment.center,
-        ),
-        SizedBox(
-          height: 20,
-        ),
-        Align(
-          child: RichText(
-            textAlign: TextAlign.center,
-            text: new TextSpan(
-              // Note: Styles for TextSpans must be explicitly defined.
-              // Child text spans will inherit styles from parent
-              style: new TextStyle(
-                fontSize: 14.0,
-                color: Colors.black,
-              ),
-              children: <TextSpan>[
-                new TextSpan(
-                    text: 'By responding, I accept the\n'
-                        ''),
-                new TextSpan(
-                    text: 'Terms and Conditions ',
-                    recognizer: new TapGestureRecognizer()
-                      ..onTap = () {
-                        launchWebUrl(TERMS_AND_CONDITIONS);
-                      },
-                    style: new TextStyle(fontWeight: FontWeight.bold)),
-                new TextSpan(text: ' and '),
-                new TextSpan(
-                    text: ' Privacy Policy.',
-                    recognizer: new TapGestureRecognizer()
-                      ..onTap = () {
-                        launchWebUrl(PRIVACY_POLICY);
-                      },
-                    style: new TextStyle(fontWeight: FontWeight.bold)),
-              ],
-            ),
-          ),
-          alignment: Alignment.center,
-        ),
-        SizedBox(
-          height: 20,
-        ),
-      ],
-    );
+  Widget firstCardContent() {
+    Widget questionPair = CTQuestionPair(
+        positiveQuestionText: "I am not infected with Coronavirus",
+        negativeQuestionText: "I am infected with Coronavirus",
+        onNegativeQuestionClick: () async {
+          dialogOnResponse(2);
+          FirestoreRepository.setUserSeverity(1);
+        },
+        onPositiveQuestionClick: () async {
+          dialogOnResponse(2);
+          FirestoreRepository.setUserSeverity(0);
+        });
+
+    return CTBottomSheetWidget(
+        mainQuestionText: "Q. How are you feeling today?",
```

Comment:
@SamIAmHarris Can we pull these i18 for localisations ?
https://flutter.dev/docs/development/accessibility-and-localization/internationalization